### PR TITLE
Make the network tests compatible with the environment moved to rush

### PR DESCRIPTION
**Pull request by michalseweryn**

### CODE REVIEW

File: finality-aleph/src/network/tests.rs [hunk 1]
```diff
@@ -0,0 +1,423 @@
...
+            oneshot_sender: self.oneshot_sender.clone(),
+            report_peer: self.report_peer.clone(),
+            disconnect_peer: self.disconnect_peer.clone(),
+            send_message: self.send_message.clone(),
+            announce: self.announce.clone(),
+            add_set_reserved: self.add_set_reserved.clone(),
+            remove_set_reserved: self.remove_set_reserved.clone(),
+        }
+    }
+}
```
Comment (DamianStraszak) [inline finality-aleph/src/network/tests.rs:61]: Deriving Clone would not work here?

File: finality-aleph/src/network/tests.rs [hunk 2]
```diff
@@ -0,0 +1,423 @@
...
+            match data.network.send_message.1.lock().next().await {
+                Some((peer_id, protocol, message)) => {
+                    peer_ids.insert(peer_id);
+                    assert_eq!(protocol, PROTOCOL_NAME);
+                    match <(SessionId, Vec<u8>)>::decode(&mut message.as_slice()) {
+                        Ok((session_id, message)) => {
+                            assert_eq!(session_id.0, 0);
+                            assert_eq!(message, fake_message);
+                        }
+                        _ => panic!("Expected a request parents consensus message"),
```
Comment (DamianStraszak) [inline finality-aleph/src/network/tests.rs:348]: This panic message does not sound right, probably some copy-paste issue. Happens also below couple of times.

File: finality-aleph/src/network/tests.rs (hunk 1 above)
Comment (michalseweryn) [reply finality-aleph/src/network/tests.rs:61]: Nope :/

File: finality-aleph/src/network/tests.rs (hunk 2 above)
Comment (michalseweryn) [reply finality-aleph/src/network/tests.rs]: Oops... fixed that

File: finality-aleph/src/network/tests.rs [hunk 3]
```diff
@@ -0,0 +1,425 @@
...
+            oneshot_sender: self.oneshot_sender.clone(),
+            report_peer: self.report_peer.clone(),
+            disconnect_peer: self.disconnect_peer.clone(),
+            send_message: self.send_message.clone(),
+            announce: self.announce.clone(),
+            add_set_reserved: self.add_set_reserved.clone(),
+            remove_set_reserved: self.remove_set_reserved.clone(),
+        }
+    }
+}*/
```
Comment (DamianStraszak) [inline finality-aleph/src/network/tests.rs:63]: Can get rid of that commented out piece :)

Comment (michalseweryn) [reply finality-aleph/src/network/tests.rs]: Done

**Pull request merged**